repair/repair.cc

```diff
@@ -1218,13 +1217,12 @@ struct repair_options {
             options.erase(it);
         }
     }
-
+public:
     // A range is expressed as start_token:end token and multiple ranges can
     // be given as comma separated ranges(e.g. aaa:bbb,ccc:ddd).
-    static void ranges_opt(dht::token_range_vector& var,
-            std::unordered_map<sstring, sstring>& options,
-                        const sstring& key) {
-        auto it = options.find(key);
+    void set_ranges(std::unordered_map<sstring, sstring>& options,
```

Comment:
why do you want to rename? ranges_opt, bool_opt, list_opt are parsing the options. The existing names are consistent. More importantly, this patch does not do what the subject say. Let's not add cosmetic changes and make the changes to other subsystems minimal.

Reply:
This change is gone once I moved the check to repair_info constructor.